SUPPORTED_PLATFORMS.md

```diff
@@ -0,0 +1,31 @@
+# Supported platforms
+
+|  System | Support type | Supported versions | Notes |
+|---|---|---|---|
+| GNU/Linux | Tier 1 | Linux >= 2.6.18 with glibc >= 2.5 | |
+| macOS | Tier 1 | macOS >= 10.7 | |
+| Windows | Tier 1 | Windows >= XP SP1 | MSVC 2008 and later are supported |
+| FreeBSD | Tier 1 | >= 9 (see note) | |
+| AIX | Tier 2 | >= 6 | Maintainer: @iWuzHere |
+| z/OS | Tier 2 | >= V2R2 | Maintainer: @jBarz |
+| Linux with musl | Tier 2 | musl >= 1.0 | |
+| SunOS | Tier 3 | Solaris 121 and later | |
+| MinGW | Tier 3 | MinGW 32 (from mingw.org) and MinGW-w64 | |
+| Other | Tier 3 | N/A | |
+
+#### Note on FreeBSD 9
+
+While FreeBSD is supported as Tier 1, FreeBSD 9 will get Tier 2 support until
+it reached EOL, in December 2016.
```

Comment:
s/reached/reaches/ and maybe replace 'EOL' with 'end of life' for people not familiar with the shorthand.